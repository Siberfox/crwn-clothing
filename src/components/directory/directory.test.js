import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";

import { Directory } from "./directory.component";

it("should render Directory component", () => {
  const mockSections = [{ id: "idkfa" }, { id: "iddqd" }];
  const wrapper = shallow(<Directory sections={mockSections} />);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});
