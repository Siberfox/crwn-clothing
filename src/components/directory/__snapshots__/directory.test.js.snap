// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Directory component 1`] = `
<styled.div>
  <withRouter(MenuItem)
    key="idkfa"
  />
  <withRouter(MenuItem)
    key="iddqd"
  />
</styled.div>
`;
